browser-test/src/support/admin_questions.ts

```diff
@@ -550,6 +551,37 @@ export class AdminQuestions {
     enumeratorName = AdminQuestions.DOES_NOT_REPEAT_OPTION,
     exportOption = AdminQuestions.NO_EXPORT_OPTION,
   }: QuestionParams) {
+    await this.createCheckboxQuestion({
```

Comment:
Note to the reviewer: Having distinct `add`/`create` methods here follows a pre-existing pattern (e.g `createDropdownQuestion`)